premerge/windows_2022_runner_values.yaml

```diff
@@ -0,0 +1,48 @@
+githubConfigUrl: "https://github.com/llvm"
+githubConfigSecret: "github-token"
+
+minRunners: 0
+maxRunners: 16
+
+runnerGroup: ${ runner_group_name }
+
+template:
+  metadata:
+    annotations:
+      cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
+  spec:
+    tolerations:
+    - key: "node.kubernetes.io/os"
+      operator: "Equal"
+      value: "windows"
+      effect: "NoSchedule"
+    - key: "premerge-platform"
+      operator: "Equal"
+      value: "windows-2022"
+      effect: "NoSchedule"
+    nodeSelector:
+      premerge-platform: windows
```

Comment:
Shouldn't this value be "windows-2022"?

Reply:
Yep. I just modified the taint and thought I was done...